src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -253,9 +254,18 @@ internal static WorkerInitResponse WorkerInitRequestHandler(WorkerInitRequest re
             var response = new WorkerInitResponse
             {
                 Result = new StatusResult { Status = StatusResult.Types.Status.Success },
-                WorkerVersion = WorkerInformation.Instance.WorkerVersion
+                WorkerVersion = WorkerInformation.Instance.WorkerVersion,
+                WorkerMetadata = new WorkerMetadata
+                {
+                    RuntimeName = RuntimeInformation.FrameworkDescription,
```

Comment:
Looks like WorkerInformation instance exposes some of these props. Ex: `WorkerInformation.Instance.FrameworkDescription`. It also has an `OSArchitecture` property which returns `RuntimeInformation.OSArchitecture`

Reply:
Yes. All of those end up forwarding. For this, I don't think there's a lot of value in capturing the `OSArchitecture`, as that is already known, but we're capturing the `ProcessArchitecture`